packages/docid/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "dependencies": {
     "cids": "1.0.2",
-    "multibase": "3.0.1",
```

Comment:
@oed, is it important that the version we have now does not have the leading `^`?  Doesn't that mean we wanted to pin to this specific version for some reason?  The tests pass, should I go ahead and merge this?

Reply:
I don't know the reason here. Maybe @zachferland knows.